src/pages/main-site/Events.js

```diff
@@ -6,12 +6,12 @@ import Footer from "../../components/main-site/Footer";
 import EventsCalendar from "../../components/main-site/EventsCalendar";
 
 import "../../styles/main-site/main.css";
-
 const Members = () => {
   return (
     <div className="landing">
       <Header />
-
+      <div style={{padding:"20px"}}></div>
```

Comment:
Same with this div (if we need a spacer component we should create one, but for these cases we can just apply margin to the components that need it)